feeds/wifi-trunk/hostapd/files/hostapd.sh

```diff
@@ -321,12 +332,32 @@ hostapd_set_psk() {
 	for_each_station hostapd_set_psk_file ${ifname}
 }
 
+append_roaming_consortium() {
+	[ -n "$1" ] && append bss_conf "roaming_consortium=$1" "$N"
+}
```

Comment:
nitpick, might make sense to have a generic call back that you pass the parameter to as $2
		json_for_each_item "append_list_param" "roaming_consortium" "roaming_consortium"
￼